networkit/helpers.pyx

```diff
@@ -41,18 +53,60 @@ def ranked(sample):
 	return result
 
 def sorted(sample):
```

Comment:
I think the idea is to use the sort-functionality from C++ (`auxiliary/Parallel.hpp`). Haven't checked that, but it should be faster in most cases?

Reply:
This uses `std::sort` on a copy of the input array, and I wouldn't be so sure that this implementation is faster than using `sorted()`. I think we don't need this function because `sorted()` in Python is already pretty fast, and we don't have benchmark data that shows the benefit of this additional function.